<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apply_to</class>
 <widget class="QDialog" name="apply_to">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apply this to</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="form_area">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="condition_label">
        <property name="text">
         <string>Condition</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="condition_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Same</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Different</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Value</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="condition_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Same</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Different</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Value</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="criterion_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sample ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Process Order</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Type</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="value_1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="criterion_1">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sample ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Process Order</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Type</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="value_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="condition_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Same</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Different</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Value</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="value_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="criterion_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sample ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Process Order</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Type</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="value_label">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="criterion_label">
        <property name="text">
         <string>Criterion</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="criterion_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sample ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Process Order</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data Type</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="value_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="condition_1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>&lt;&lt; select &gt;&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Same</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Different</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Value</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_area">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontal_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_apply_to_all">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Apply to all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_accept">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_cancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
